extensions/amp-google-assistant-assistjs/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ychsieh'}, {name: 'ianba'}],
```

Comment:
Please have @ianba register their Google and Github accounts at go/github

Reply:
Done. Let me remove Ian for now until he finishes the account linking.